apps/redi-talent-pool/src/components/organisms/Avatar.tsx

```diff
@@ -84,8 +109,12 @@ const AvatarEditable = ({
   const [imageSrc, setImageSrc] = useState(null)
   const [imageFileName, setImageFileName] = useState('')
 
+  const [isUploading, setIsUploading] = useState(false)
+  const [uploadProgress, setUploadProgress] = useState(0)
+
   const [crop, setCrop] = useState({ x: 0, y: 0 })
   const [zoom, setZoom] = useState(1)
+  const [minZoom, setMinZoom] = useState(1)
 
   // This is how to keep functions in React state: https://stackoverflow.com/questions/55621212/is-it-possible-to-react-usestate-in-react
   const [nextFn, setNextFn] = useState(() => (croppedImgFile) => {
```

Comment:
Seems like a strange way to store a function, isn't `useRef` better for this use case?

Reply:
Yeah, haven't noticed this part as it was out of the scope of this work but definitely I'll update it.